build/config/compiler/BUILD.gn

```diff
@@ -63,12 +63,13 @@ config("compiler") {
   # not be compatible with newer ones. To achieve this, we insert a synthetic
   # define into the compile line. We only do this for toolchains that we use
   # from buildtools.
-  toolchain_version = exec_script("//build/util/lastchange.py",
-                                  [
-                                    "--source-dir=" + rebase_path("//buildtools"),
-                                    "--revision-only",
-                                  ],
-                                  "trim string")
+  toolchain_version =
```

Comment:
The changes in this file appear to be only reformattings.  Can you revert this?

Reply:
This is the result of a GN format that the previous committer had not performed. Do you want me to submit the change in a different patch or not make it at all?

Reply:
Done. PTAL.